examples/use_caching.py

```
@@ -0,0 +1,306 @@
+    cache_get,
+    cache_set,
+    cache_delete,
+    cache_clear,
+    get_cache_stats,
+    reset_cache_stats,
+    cached,
+    preload_cache,
+    watch_file,
+    stop_file_watcher,
+    CacheBackend,
+    CachePolicy,
```

Ruff (0.8.2)

`pyprocessor.utils.core.cache_manager.CachePolicy` imported but unused

Remove unused import: `pyprocessor.utils.core.cache_manager.CachePolicy`

(F401)